code/game/objects/items/control_wand.dm

```diff
@@ -181,10 +181,15 @@
 	item_state = "hacktool"
 	var/hack_speed = 1.5 SECONDS
 	var/busy = FALSE
+	/// How far can we use this. Leave `null` for infinite range
+	var/range
 
 /obj/item/door_remote/omni/access_tuner/afterattack(obj/machinery/door/D, mob/user)
 	if(!istype(D, /obj/machinery/door/airlock) && !istype(D, /obj/machinery/door/window))
 		return
+	if(range && get_dist(src, D) > range)
```

Comment:
Just a teensy bit more explicit
```suggestion
	if(!isnull(range) && get_dist(src, D) > range)
```

Reply:
When would this matter?

Reply:
It's not going to change any behavior but it looks like you're trying to use null as more of a sentinel, so it might not be a bad idea to make it clear that's what we're looking for.